src/views/Activity.vue

```diff
@@ -91,158 +98,129 @@ export default {
       duration: "",
       eventcount: 0,
       errormsg: "",
+      numberOfWindowTitles: 5
     }
   },
 
   watch: {
     '$route': function(to, from) {
       console.log("Route changed")
       this.query();
-    },
-
-    'errormsg': function(to, from){
-      console.log(to);
     }
   },
 
   computed: {
-    readableDuration: function() {
-      return time.seconds_to_duration(this.duration);
-    },
-    host: function() {
-      return this.$route.params.host;
-    },
-    date: function() {
-      return this.$route.params.date || new Date().toISOString();
-    },
-    datestart: function() {
-      // Returns a
-      console.log(this.date);
-      let datestart = moment(this.date).startOf('day').format();
-      console.log(datestart);
-      return datestart;
-    },
-    dateShort: function() {
-      return moment(this.date).format("YYYY-MM-DD");
-    }
-
+    readableDuration: function() { return time.seconds_to_duration(this.duration) },
+    host: function() { return this.$route.params.host },
+    date: function() { return this.$route.params.date || new Date().toISOString() },
+    dateStart: function() { return moment(this.date).startOf('day').format() },
+    dateShort: function() { return moment(this.date).format("YYYY-MM-DD") },
+    windowBucketId: function() { return "aw-watcher-window" + (PRODUCTION ? "_" : "-testing_") + this.host },
+    afkBucketId:    function() { return "aw-watcher-afk"    + (PRODUCTION ? "_" : "-testing_") + this.host }
   },
 
   mounted: function() {
-    // Create summary
-    var summary_elem = document.getElementById("appsummary-container")
-    summary.create(summary_elem);
-
-    // Create timeline
-    var timeline_elem = document.getElementById("apptimeline-container")
-    timeline.create(timeline_elem);
-
-    $Info.get().then(
-      (response) => {
-        this.query();
-      },
-      (response) => {
-        this.errormsg = "Request error "+response.status+" at get info. Server offline?";
-      }
-    );
+    summary.create(document.getElementById("appsummary-container"));
+    summary.create(document.getElementById("windowtitles-container"));
+    timeline.create(document.getElementById("apptimeline-container"));
+
+    this.query();
   },
 
   methods: {
-    previousDay: function() {
-        return moment(this.datestart).subtract(1, 'days').format("YYYY-MM-DD");
-    },
-    nextDay: function() {
-        return moment(this.datestart).add(1, 'days').format("YYYY-MM-DD");
+    previousDay: function() { return moment(this.dateStart).subtract(1, 'days').format("YYYY-MM-DD") },
```

Comment:
I just got annoyed by the long file and felt that it could be onelined since you should never really need to read it. Might be better to turn it into a computed property.